modules/bash-commons/src/os.sh

```diff
@@ -36,6 +36,10 @@ function os_is_redhat {
   grep -q "Red Hat Enterprise Linux Server release $version" /etc/*release
 }
 
+# Returns true (0) if this is a WSL kernel or false (1) otherwise.
+function os_is_wsl_kernel {
+  [[ $(uname -r | tail -c19) == "microsoft-standard" ]]
```

Comment:
How stable / reliable is looking at column 19? Could that change in a future release?

Reply:
This does seem brittle... Maybe use `grep` instead?

Reply:
Bash supports regex, no need to grep, /etc/os-release would be Ubuntu/Redhat etc, whatever WSL2 container the user has loaded. It's a config string in their build, https://github.com/microsoft/WSL2-Linux-Kernel/blob/master/Microsoft/config-wsl#L22